typings/entry.d.ts

```diff
@@ -18,20 +18,20 @@ export interface EntrySys extends MetaSysProps {
   publishedCounter?: number,
 }
 
-export interface Entry
-  extends EntryProp,
+export interface Entry<TEntryFields = any>
```

Comment:
Any reason why `TEntryFields` is `any` seeing as `EntryProp` has an explicit key-value type defined?

Some suggestions
- create a new type like:
```
type KVMap = {
  [key: string]: any
}
```

- update EntryProp
```
export interface EntryProp<TFields> { ...
```

- update Entry interface
```
export interface Entry<TEntryFields = KVMap> {
  extends EntryFields<TEntryFields>,
  DefaultElements<EntryFields<TEntryFields> & EntrySys>,
    MetaSys<EntrySys> {
  archive(): Promise<Entry<TEntryFields>>,
  delete(): Promise<void>,
  getSnapshot(id: string): Promise<any>,
  getSnapshots(): Promise<Collection<any>>,
  isArchived(): boolean,
  isDraft(): boolean,
  isPublished(): boolean,
  isUpdated(): boolean,
  publish(): Promise<Entry<TEntryFields>>,
  unarchive(): Promise<Entry<TEntryFields>>,
  unpublish(): Promise<Entry<TEntryFields>>,
  update(): Promise<Entry<TEntryFields>>,
}
```

Reply:
my understanding was that if it wasn't an `any` it wouldn't be backwards compatible, but I  like your type suggestion, so I can certainly update that one.